### PR TITLE
fs: add chmod utility

### PR DESCRIPTION
**Pull request by patriki01**
Added method to perform chmod.
Feel free to suggest changes. 🤖

PS: I accidentally deleted the original branch, so the [original PR](https://github.com/next-actions/pytest-mh/pull/17) was closed.

### CODE REVIEW

**Comment by patriki01** [comment]
Hi @ikerexxe,
I discussed this with Pavel and he told me, that we can add chmod() to this framework. He also had some comments, which I applied.

File: pytest_mh/utils/fs.py [hunk 1]
```diff
@@ -527,3 +527,23 @@ def wc(
...
+        :type mode: str
+        :param path: File or folder whose permissions change
+        :type path: str
+        :param args: Additional options
+        :type args: list[str]
+        :return: Result of process
+        :rtype: SSHProcessResult
+        """
+        self.backup(path)
+        self.logger.info(f'Changing mode bits for file "{path}" on {self.host.hostname}')
```
Comment (ikerexxe) [inline pytest_mh/utils/fs.py:547]: As this can be both a file or a folder I'd avoid using the word `file`.

**Review by ikerexxe — COMMENTED** [review]
Minor comment, apart from that it looks good.

**Review by ikerexxe — APPROVED** [review]
LGTM. Thank you for the patch!

**Review by pbrezina — APPROVED** [review]

**Pull request merged**